tests/plugins/test_pull_base_image.py

```diff
@@ -661,6 +669,66 @@ def workflow_callback(workflow):
         assert pulling_msg in caplog.text
         assert tagging_msg in caplog.text
 
+    @pytest.mark.parametrize('orchestrator, fail', tuple(product((True, False), repeat=2)))
+    def test_parent_images_digests(self, caplog, orchestrator, fail):
+        def workflow_callback(workflow):
+            workflow = self.prepare(workflow)
+            if fail:
+                # fail to provide platform specific digests
+                manifest_list = {
+                    'manifests': []
+                }
+
+                manifest_tag = 'registry.example.com' + '/' + BASE_IMAGE_W_SHA
+                base_image_result = ImageName.parse(manifest_tag)
+                manifest_image = base_image_result.copy()
+
+                (flexmock(atomic_reactor.util)
```

Comment:
TODO: I have to figure out, why in fail case I'm still getting manifest_list with digests